scripts/release.js

```diff
@@ -127,9 +132,7 @@ prompt([
         const changelog = fs.readFileSync(changelogPath, 'utf8');
 
         // Add the new entry
-        const changelogEntry = `## <sub>v${newVersion}</sub>\n\n#### ${moment().format(
-          '_MMM. D, YYYY_'
-        )}`;
+        const changelogEntry = `## v${newVersion}\n\n### ${formattedDate()}`;
```

Comment:
I'm a bit lost, contextually, so I might be being stupid.  This is markdown tho, right? How come there are 2 `#` before and 3 `#` after?  Not that it is really that important!

Reply:
OMG pls ignore 🙈

Reply:
Not stupid at all, I was _also_ lost reading this code. Guessing you worked out that this is a h2 and h3. It was super confusing before going 2 to 4.